enterprise/utils.py

```diff
@@ -900,6 +900,20 @@ def get_cache_key(**kwargs):
     return get_django_cache_key(**kwargs)
 
 
+def chunk_content_keys(content_filter, chunk_size):
```

Comment:
Could we make use of the [existing `batch` util method](https://github.com/edx/edx-enterprise/blob/a149e9e55cdae56179ee470ba72ebeab3d34aa49/enterprise/utils.py#L2050) rather than redefining a new one?